appcelerator-archive/amp
# `amp user resend-verify` to resend verification email

fixes: #1644 #1646 

Adds command `amp user resend-verify` which is used to resend the verification email first sent during the `amp user signup` command to the specified users registered email address.

Updates the existing verification email html:

- Removes the "CONFIRM EMAIL ADDRESS" button as we aren't using the UI at the moment.
- Fixes the server address in the `amp verify` command call.
- Fix small typo

to test:

Use `amplifier_yml` secret with a valid SendGrid key

```
$ ampmake build
$ amp cluster create --registration=email --notifications=true
$ amp user signup --name=user --email=email@me.com --password=password
Hi user! Please check your email to complete the signup process.
$ amp user resend-verify user
The verification email has been re-sent to your address, user
```
